lib/rory/application.rb

```diff
@@ -39,7 +39,7 @@ def instance
       end
 
       def root=(root_path)
-        @root = Pathname.new(root_path).expand_path
+        $:.unshift @root = Pathname.new(root_path).realpath
```

Comment:
I'm torn on this one.  I'm all for using `#realpath`; this was just a failure on my part to know the `Pathname` lib well enough, and also a failure to test this out in symlinked directories.

As for adding root to the loadpath - I'm not as convinced.  Personally I prefer `require_relative` - it still exhibits performance benefits, for obvious reasons, though since 2.x it's not so noticeable with a smaller loadpath.  What I *really* like is the effect on file organization and understanding thereof, and the reduced likelihood of conflicts between libraries.

Of course, I'm open to being convinced otherwise.  Convince me. :smile: